packages/react-next/src/components/Toggle/Toggle.types.ts

```diff
@@ -183,3 +203,26 @@ export interface IToggleStyles {
    */
   text: IStyle;
 }
+
+/**
+ * {@docCategory Toggle}
+ */
+export interface IToggleSlots {
```

Comment:
as discussed, let's not expose any slot related typing yet. including the shorthands in ComponentProps